.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = tab
```

Comment:
i think this should be spaces?

Reply:
Ehh... no problem, except that the code is currently indented with tabs.

Reply:
question is asked in discord :)

Reply:
Done.